packages/cli/src/commands/secrets/index.js

```diff
@@ -142,9 +48,11 @@ const main = async client => {
   try {
     ({ contextName } = await getScope(client));
   } catch (err) {
-    if (err.code === 'NOT_AUTHORIZED' || err.code === 'TEAM_DELETED') {
```

Comment:
Not strictly required here but TS will complain in the future and this is how other commands wrap API errors